### PR TITLE
fix: pairwise sig for MR x MR (w/ overlaps)

### PR DESCRIPTION
**Pull request by slobodan-ilic**

### CODE REVIEW

File: src/cr/cube/matrix/cubemeasure.py [hunk 1]
```diff
@@ -197,10 +197,19 @@ def factory(cls, cube, dimensions, slice_idx):
...
+                dimensions,
+                cube.overlaps[cls._slice_idx_expr(cube, slice_idx)],
+                cube.valid_overlaps[cls._slice_idx_expr(cube, slice_idx)],
+            )
+            if dimension_types == (DT.MR, DT.MR)
+            else _CatXMrOverlaps(
+                dimensions,
+                cube.overlaps[cls._slice_idx_expr(cube, slice_idx)],
+                cube.valid_overlaps[cls._slice_idx_expr(cube, slice_idx)],
+            )
```
Comment (ernestoarbitrio) [inline src/cr/cube/matrix/cubemeasure.py:212]: I would use a variable like `slice_idx_expr = cls._slice_idx_expr(cube, slice_idx)`, and use it in the 4 slicing operations at line 204,205,210,211

File: src/cr/cube/matrix/cubemeasure.py [hunk 2]
```diff
@@ -248,6 +257,16 @@ def valid_overlaps(self):
...
+class _MrXMrOverlaps(_BaseCubeOverlaps):
+    @lazyproperty
+    def overlaps(self):
+        return self._overlaps[:, 0, :, 0]
+
+    @lazyproperty
+    def valid_overlaps(self):
+        return np.sum(self._valid_overlaps[:, 0, :, 0:2], axis=2)
+
+
```
Comment (ernestoarbitrio) [inline src/cr/cube/matrix/cubemeasure.py:269]: docstrings missed

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
Some minor comments left, but my major concern is that new `mr-x-mr` fixture with 5k lines, that huge, it there a chance to groom that file or use a smaller one?

File: src/cr/cube/matrix/cubemeasure.py (hunk 2 above)
Comment (slobodan-ilic) [reply src/cr/cube/matrix/cubemeasure.py]: yeah, this was updated even prior to reading comments

**Pull request merged**